web/templates/modal_banner/unsubscribed_participants_warning_banner.hbs

```diff
@@ -0,0 +1,22 @@
+{{#> modal_banner . }}
+    <p class="banner_message">
+        {{#if few_unsubscribed_participants}}
+            {{#tr}}Some topic participants are not subscribed to{{/tr}}<br/>
```

Comment:
This translation syntax doesn't work for languages with a different word order than English. See
https://zulip.readthedocs.io/en/latest/translating/internationalization.html to learn a lot more about this. But you'll want to use the `{{#*inline "z-user"}` class of syntax here to be able to in include the channel and participants here.

I think you also want to use `inline_decorated_stream_name`.

Reply:
I can see `{{#*inline "z-user"}` only used for a single username

Reply:
I think it's `{{#*inline "z-user-names"}}`